src/main/ui/src/App.js

```diff
@@ -1,3 +1,16 @@
+import SearchByState from './components/SearchByState';
```

Comment:
This doesn't look right. When I open up vscode it says 
`Parsing error: Identifier 'App' has already been declared. (20:6)`

Reply:
this is what was causing the search to be repeated.. it was doubled in index.js